java/arcs/android/storage/service/BindingContextStats.kt

```diff
@@ -21,7 +32,8 @@ interface BindingContextStatisticsSink {
  *
  * **Note:** This makes the assumption that the distribution of processing times is normal.
  */
-interface BindingContextStatistics : BindingContextStatisticsSink {
+interface BindingContextStatistics :
+    BindingContextStatisticsSink {
```

Comment:
Same line?

Reply:
Ah crap, Refactor->Move loves to try to help out by reformatting, and it ends up wrapping too aggressively.